docs/content/docs/start/examples.md

```diff
@@ -17,3 +17,45 @@ The sticky notes [brainstorming app](https://github.com/microsoft/FluidExamples/
 shows how simple experiences become compelling when you make them collaborative. This example uses distributed
 maps (SharedMaps) to update the state of sticky notes as well as keep track of who added which ideas and who
 voted for those ideas.
+
+## Collaborative text area
+
+The [collaborative text area app](https://github.com/microsoft/FluidExamples/tree/main/angular-demo) shows
```

Comment:
Did you want to point to here: https://github.com/microsoft/FluidExamples/tree/main/collaborative-text-area ? Maybe it would be worthwhile to double check other links/titles on this page as well.

Reply:
good catch. fixed.